modules/ROOT/pages/graph-analytics/index.adoc

```diff
@@ -19,19 +22,36 @@ The Graph Analytics plugin allows you to use the link:https://neo4j.com/docs/gra
 * 4 GB of memory or more
 * Any supported cloud providers and regions
 
-You can enable or disable the plugin when necessary.
+You can enable or disable Graph Analytics when necessary.
+The plugin shares compute and memory resources with the AuraDB server, so you do not incur any additional costs when you enable it.
+
+[[aura-gds-serverless]]
+== Graph Analytics Serverless
+
+include::partial$EAP.adoc[]
+
+Graph Analytics Serverless allows you to use the Graph Data Science library regardless of where your source data is stored.
+
+Graph Analytics Serverless runs in Aura as a link:https://neo4j.com/docs/graph-data-science-client/current/gds-session/[dedicated service] optimised for analytics workloads, with no memory or compute resources shared with your data store.
+
+=== From an AuraDB instance
+
+label:AuraDB-Business-Critical[] label:AuraDB-Virtual-Dedicated-Cloud[]
 
-[WARNING]
-====
-You can no longer use the plugin if you resize your instance to less than 4 GB of memory.
-====
+You can use Graph Analytics Serverless from any AuraDB Business Critical or AuraDB Virtual Dedicated Cloud instance with the following requirements:
+
+* Neo4j version 5 or later
+* Any supported cloud providers and regions
 
 [[aura-ds]]
 == AuraDS
 
 label:AuraDS-Professional[] label:AuraDS-Enterprise[]
 
-AuraDS is the fully managed version of the link:https://neo4j.com/docs/graph-data-science/current/introduction/[Graph Data Science library].
+AuraDS is the fully managed version of the Graph Data Science library where the <<aura-gds-plugin>> is deployed by default.
```

Comment:
I think this sentence reads too strongly in favour of AuraDS (my bias is negative here).
```suggestion
AuraDS is a database service similar to AuraDB centered around the Graph Data Science library where the <<aura-gds-plugin>> is always installed.
```